Editor/Resources/EditorWindow/common.uss

```diff
@@ -0,0 +1,14 @@
+﻿/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+:root {
+    height: 100%;
+    width: 100%;
```

Comment:
What does putting these 100% in :root do for everything that pulls in this common file?

Reply:
true, I guess that should remain in `GameLiftPlugin.uss`

Reply:
Removed from `common.uss` and replaced in `GameLiftPlugin.uss`